server/pbench/bin/state/config-satellite/pbench-server.cfg

```diff
@@ -0,0 +1,38 @@
+[DEFAULT]
+# Unit test defaults
+unittest-dir = /var/tmp/pbench-test-server
+default-deploy-dir = %(unittest-dir)s/opt/pbench-server-satellite
+default-host = pbench-satellite.example.com
+
+###########################################################################
+## Deployment section
+###########################################################################
+[pbench-server]
+environment = unit-test
+pbench-top-dir = %(unittest-dir)s/pbench-satellite
+pbench-local-dir = %(unittest-dir)s/pbench-satellite-local
+# A satellite just unpacks and syncs
+dispatch-states = TO-UNPACK, TO-SYNC
+# A satellite just needs to prep and process results, no backup necessary
+roles = pbench-prep, pbench-results
+use-mail-for-status = yes
+realhost = pbench.example.com
```

Comment:
Do we need an explanation here? Why is `realhost` required?

Reply:
Added a comment explaining why it is needed.